sdk/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hc_portal_sdk"
```

Comment:
Downstream projects that depend on this crate expect it to be published on crates.io. This has not been done yet due to my lack of permissions to publish them. At the moment, the depending crates use git urls for this crate in their `Cargo.toml` files

Reply:
Please could you start a thread about this (or tag me and Zippy in it if there's already one) and make sure this gets progressed?